inst/rmarkdown/templates/springer_article/skeleton/bibliography.bib

```diff
@@ -0,0 +1,233 @@
+%% This BibTeX bibliography file was created using BibDesk.
+%% http://bibdesk.sourceforge.net/
```

Comment:
Could you only keep the bib entries that are actually used in the example article?

Reply:
done